db/schema.rb

```diff
@@ -658,6 +660,101 @@
      FROM (appointments ap
        JOIN facilities f ON ((ap.facility_id = f.id)));
   SQL
+  create_view "latest_blood_pressures_per_patient_per_days", materialized: true, sql_definition: <<-SQL
```

Comment:
Can we skip committing these schema reordering chunks?

Reply:
Yup I'll clean up the schema 👍